spec.md

```diff
@@ -3938,6 +3952,27 @@ The shift right operation result is the arbitrary precision signed integer arith
 e2 sign bits from e1 are shifted into the most significant bits, and the e2 least significant bits of e1 are truncated.
 e2 must be non-negative.
 
+## List Operations
+
+List operations create `List`{.firrtl} property type expressions from other property expressions.
```

Comment:
Formally speaking, what is a "property expression"?

Reply:
These are introduced in the section Primitive Property Operations.